### PR TITLE
Test Generator for SgfParsinf Exercise and Example updated. Issue #635

### PR DESCRIPTION
**Pull request by deniscapeto**
I added generator classe for SgfParsinf exercise and updated the example file. All tests are passing.

### CODE REVIEW

File: generators/Exercises/Generators/SgfParsing.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            if (testMethod.Expected is Dictionary<string, object> &&
+                (testMethod.Expected as Dictionary<string, object>).ContainsKey("error"))
+                testMethod.ExceptionThrown = typeof(ArgumentException);
+            else
+                testMethod.Expected = RenderTree(testMethod.Expected);
+
+            testMethod.TestedClass = "SgfParser";
+            testMethod.TestedMethod = "ParseTree";
+
+            testMethod.UseVariablesForInput = true;
```
[the remark this answers is not in the document]
Comment (deniscapeto) [reply generators/Exercises/Generators/SgfParsing.cs]: Sure. Done

[the remark this answers is not in the document]
Comment (ErikSchierboom) [reply generators/Exercises/Generators/SgfParsing.cs]: Ehm, did you re-run the generator?

[the remark this answers is not in the document]
Comment (deniscapeto) [reply generators/Exercises/Generators/SgfParsing.cs]: uh. I forgot to push the test file. Sorry. It's updated now.

**Comment by ErikSchierboom** [comment]
Merged, thanks!

**Pull request merged**